TowerDefence_UnityProject/Assets/Scripts/TowerDefence/World/Path/PathLineRenderer.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+using Zenject;
+
+namespace TowerDefence.World.Path
+{
+    public class PathLineRenderer : MonoBehaviour
```

Comment:
Add a abstract underlying component. So this part can be easily replaced at a later point if that is required